includes/class-wc-payments-session-service.php

```diff
@@ -16,6 +16,8 @@
  */
 class WC_Payments_Session_Service {
 
+	const SESSION_STORE_ID_OPTION = 'woopayments_session_store_id';
```

Comment:
Same comment/question as earlier about whether we want to keep the `wcpay_` prefix for all options for now, or update them all to `woopayments_` at some point. 🙂

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/7520/commits/b35db35ae072115e60ff6242f89dc54fa4b6afb9